go/vt/vtgate/safe_session.go

```diff
@@ -124,6 +189,8 @@ func (session *SafeSession) Reset() {
 	}
 	session.mu.Lock()
 	defer session.mu.Unlock()
+	session.mustRollback = false
```

Comment:
Is this `mustRollback` change a regression or somehow necessary only due to the autocommit changes?

Reply:
I just added it for future-proofing (resetting all members to zero).